package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "neomad.org",
+  "description": "Neomad is a open-source community project for nomad workers and people willing to _nomadize_ themselves.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "babel": "babel",
```

Comment:
well, then you you have in your scripts:
```
…
"build": "webpack",
…
```
Because `npm build` is a conventional command and the developer does not have to guess what type of compiler and options he's supposed to run.
With `npm build`, it just builds. If the dev wants to know how, he knows he just looks up for this command in the package.json file and read the magic behind it.